docs/upgrade.md

```diff
@@ -90,11 +90,28 @@ process, for example:
 
 # Upgrading to v1.68.0
 
-As announced in the upgrade notes for v1.67.0, Synapse now requires a SQLite
-version of 3.27.0 or higher if SQLite is in use and source checkouts of Synapse
-now require a recent Rust compiler.
+Two changes announced in the upgrade notes for v1.67.0 have now landed in v1.68.0.
 
-Installations using 
+## SQLite version requirement
+
+Synapse now requires a SQLite version of 3.27.0 or higher if SQLite is configured as
+Synapse's database.
+
+Installations using
+
+- Docker images [from `matrixdotorg`](https://hub.docker.com/r/matrixdotorg/synapse),
+- Debian packages [from Matrix.org](https://packages.matrix.org/), or
```

Comment:
Do we actually have a dependency requirement on SQLite in Debian packages? I've failed to find one but then I'm not very fluent in Debian packaging.

Reply:
No, but we only release Debian packages for recent distros with sufficiently new SQLites.

Reply:
See https://github.com/matrix-org/synapse/issues/12983#issuecomment-1149828274 and https://github.com/matrix-org/synapse/issues/12983#issuecomment-1238479218

Reply:
Ah I see. Though one could nitpick by saying it would be technically possible to run on a distro that used to package a pre-3.27 SQLite, and the system admin has updated their Synapse but never their SQLite.

Reply:
> See https://github.com/matrix-org/synapse/issues/12983#issuecomment-1149828274 and https://github.com/matrix-org/synapse/issues/12983#issuecomment-1238479218

Ah I think that proves my point wrong actually, ignore me then!